test/test_tensor.h

```diff
@@ -926,14 +926,25 @@ TEST(tensor, nd1) {
 
 }
 
-TEST(tensor, print) {
-    Tensor<float_t, 4> t({3,2,2,2});
+template<size_t T> std::ostream& print_tester(std::ostream &os) {
+    os << "\nPrinting Tensor of size " << T << ":\n\n";
+    std::vector<size_t> tmp(T, 2);
+    tmp.back() = 3;
 
+    Tensor<float_t, T> t(tmp);
     t.fill(float_t(1.0));
```

Comment:
I personally don't care but @edgarriba dislikes use of C-style casts.
So we can rewrite it in this way.
```
t.fill(float_t{1.0});
```